src/styles/book_list.css

```diff
@@ -0,0 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab');
+
+ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 1rem;
+  margin: 2.5rem 0;
+}
+
+li {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+  align-self: stretch;
+  padding: 2rem 7.5vw 1.75rem 2vw;
+}
+
+.info-name {
+  margin-right: auto;
+}
+
+.category {
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+  opacity: 0.5;
+  margin-bottom: 0.25rem;
+}
+
+.title {
+  height: 1.8rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.auth {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  height: 1.2rem;
+  line-height: 1.2rem;
+  margin-bottom: 0.9rem;
+  opacity: 0.8;
+}
+
+.btn {
+  height: 1.5rem;
+  border: none;
+  background-color: transparent;
+  padding: 0 1vw;
+  line-height: normal;
+  margin-bottom: 0;
+}
+
+.first-btn {
+  padding-left: 0;
+}
+
+.btn:hover {
+  cursor: pointer;
+}
+
+.info-name .btn:nth-child(odd) {
+  border-left: 1px solid #e8e8e8;
+  border-right: 1px solid #e8e8e8;
+}
+
+.percentage {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 5vw;
+}
+
+.ring {
+  transform: rotate(-90deg);
+  margin-right: 0.7vw;
+}
+
+svg {
+  width: 5.6rem;
+  height: 5.6rem;
+}
+
+svg circle {
+  fill: none;
+  stroke-width: 4.5;
+  stroke: #000;
+  stroke-dasharray: 440;
+  stroke-dashoffset: 440;
+}
+
+svg circle:nth-child(1) {
+  stroke-dashoffset: 0;
+  stroke: #e8e8e8;
+}
+
+svg circle:nth-child(2) {
+  stroke-dashoffset: calc(440 - (207 * 64) / 100);
+
+  /* stroke: linear-gradient(to bottom, #307bbe, #379cf6);   */
```

Comment:
- [ ] Kindly remove commented code from your file if it's not necessary.

Reply:
Done! 😀